### DISCUSSION TITLE
Network connectivity for Management Cluster

### DISCUSSION THREAD

**Discussion by os14** [telekom/das-schiff#17] [Q&A] [answered]
What are the requirements for network connectivity for the management clusters?

How the management cluster can use metal3 on a bare metal edge workload cluster?

As i understand metal3 need an access to the bmc of the machines on the edge, how do you achieve it if the management cluster reside in a central location?

All the edge data centers should be interconnected?

Does installing SONIC in all the network equipments solves the issue?

**Comment by schrej** [comment, marked as answer]
Yes, with the current setup, management clusters need to be able to reach the BMCs of physical servers in order to provision clusters. At the moment we have direct network connectivity.
We are not yet using edge nodes in a real-world scenario. How network connectivity will look like is not finalised.

Since SONiC is just a network operating system, installing it doesn't "solve" any connectivity issues. But it helps with automation as it provides a heterogeneous interface across different hardware.

**Reply by os14** [reply]
Thanks for the response, really appreciate the work you did in Das Schiff.

Have you started to think about solutions to the edge nodes scenario?
Can you share some ideas that you came up with?

Thanks!
